velox/exec/PrefixSortEncode.h

```diff
@@ -0,0 +1,1257 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+#include "string.h"
+#include "velox/common/base/Exceptions.h"
+
+template <size_t SIZE>
+static inline void MemcpyFixed(void* dest, const void* src) {
```

Comment:
Please, review coding guidelines and update to comply. Function and method names should start lowercase.